pkg/agent/agent.go

```diff
@@ -291,7 +291,8 @@ func (i *Initializer) initInterfaceStore() error {
 			case interfacestore.AntreaTunnel:
 				intf = parseTunnelInterfaceFunc(port, ovsPort)
 			case interfacestore.AntreaHost:
-				if port.Name == i.ovsBridge {
+				hostIfName := i.nodeConfig.UplinkNetConfig.Name
```

Comment:
Maybe we could not set "AntreaHost" type on the OVS internal port which is used as the peer for the uplink, then this case should not hit, and we don' t need to handle it, since we does nothing on such interface.

Reply:
Also think about upgrade case, that OVS port/interface br-int may also exists if the setup is upgraded from a previous version to new code?

Reply:
> Maybe we could not set "AntreaHost" type on the OVS internal port which is used as the peer for the uplink, then this case should not hit, and we don' t need to handle it, since we does nothing on such interface.

I found external node function also add "AntreaHost" type when creating host port in OVS. Do you mean it is unnecessary either and should be removed?

Reply:
> Also think about upgrade case, that OVS port/interface br-int may also exists if the setup is upgraded from a previous version to new code?

How about reusing the script in Clean-AntreaNetwork.PS1 to revert the switch to initial status if we detect any port which should not exist in new release, then applying new port by running initialize function normally?

Reply:
The port in upgrade scenario will not be renamed because it cannot be done without recreating HNSnetwork.